### PR TITLE
Make function and method docstrings comply with PEP 257

### PR DESCRIPTION
**Pull request by ricordisamoa**
https://www.python.org/dev/peps/pep-0257/

Also fix a typo.

### CODE REVIEW

File: mwoauth/flask.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def __init__(self, host, consumer_token, user_agent=None,
         @self.bp.route("/mwoauth/initiate/")
         def mwoauth_initiate():
             """
-            Starts an OAuth handshake.
```
Comment (Ladsgroup) [inline mwoauth/flask.py]: Per PEP257, on line comments should be like this """Start an OAuth handshake.""" please apply this where you are fixing them. Thanks

File: mwoauth/functions.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def initiate(mw_uri, consumer_token, callback='oob',
 def complete(mw_uri, consumer_token, request_token, response_qs,
              user_agent=defaults.USER_AGENT):
     """
-    Completes an OAuth handshake with MediaWiki by exchanging an
+    Complete an OAuth handshake with MediaWiki by exchanging an
```
Comment (ricordisamoa) [inline mwoauth/functions.py]: Maybe complete this one sometime :smile:

File: mwoauth/handshaker.py [hunk 3]
```diff
@@ -90,7 +90,7 @@ def initiate(self, callback=None):
 
     def complete(self, request_token, response_qs):
         """
-        Completes an OAuth handshake with MediaWiki by exchanging an
+        Complete an OAuth handshake with MediaWiki by exchanging an
```
Comment (ricordisamoa) [inline mwoauth/handshaker.py]: Also this one

File: mwoauth/flask.py [hunk 4]
```diff
@@ -165,16 +161,12 @@ def mwoauth_callback():
         @self.bp.route("/mwoauth/identify/")
         @authorized
         def mwoauth_identify():
-            """
-            Returns user information if authenticated
-            """
+            """Return user information if authenticated"""
```
Comment (Ladsgroup) [inline mwoauth/flask.py]: Missing period :D

File: mwoauth/functions.py (hunk 2 above)
Comment (Ladsgroup) [inline mwoauth/functions.py]: :))) @halfak ? :P

File: mwoauth/flask.py [hunk 5]
```diff
@@ -269,9 +261,7 @@ def encode_token(token):
 
 
 def _str(val):
-    """
-    Ensures that the val is the default str() type for python2 or 3
-    """
+    """Ensure that the val is the default str() type for python2 or 3"""
```
Comment (Ladsgroup) [inline mwoauth/flask.py]: Missing period.

**Pull request merged**